docs/source/getting_started/user_guide.md

```diff
@@ -1,14 +1,25 @@
 # 🖼️ User Guide
 
-At a high level, one needs to create a dataset files compatible with a streaming and then load the same dataset files using {class}`streaming.Dataset` class.
+At a very high level, one needs to convert a raw datasets into a streaming format files and then use the same streaming format files using {class}`streaming.Dataset` class for model training.
```

Comment:
convert a raw dataset into streaming format files

Reply:
Done in https://github.com/mosaicml/streaming/pull/33/commits/a634dbdbd8039c0755687d3cc8172cba15c977ed.